compute/cloud-client/pom.xml

```diff
@@ -29,6 +28,11 @@
       <artifactId>google-cloud-compute</artifactId>
       <version>1.2.0-alpha</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>1.115.0</version>
+    </dependency>
```

Comment:
Do you wish to use the Libraries-BOM?

Reply:
The samples in other languages haven't used BOM as well. Hence, to maintain consistency, I would prefer to use individual dependencies.

Reply:
Ok, though, do other languages provide a BOM?  It's supper important for Java users.